department-of-veterans-affairs/abd-vro
# Move svc-hoppy-usage into domain-xample

## What was the problem?

`svc-hoppy-usage` Gradle project was primarily an example of how to use hoppy and was not (and should not be) abiding by the same requirements we want to enforce on other projects, so we want to have it live in the `domain-xample` project.

Associated tickets or Slack threads:
- #1641  is the stream of work where i found this

## How does this fix it?[^1]

* moves `svc-hoppy-usage` out of TLD and into `domain-xample`
* moves the docker compose spec for `svc-hoppy-usage` out of the app/docker-compose.yml into domain-xample/docker-compose.yml

## How to test this PR
- `./gradlew docker build && ./gradlew :dockerComposeUp && ./gradlew :domain-xample:dockerComposeUp`

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
